### DISCUSSION TITLE
Is there any way to calculate IBUs from dry hopping or whirlpool hopping in BrewTarget?I

### DISCUSSION THREAD

**Discussion by GnuUser** [Brewtarget/brewtarget#784] [Q&A]
Is there any way to calculate IBUs from dry hopping or whirlpool hopping in BrewTarget?

I can do a zero-minute hop, but that adds zero IBUs and isn't really dry hopping. I suppose I could just add minutes to a given hop addition to "estimate" whirlpool hopping, but they're not quite the same (different temps).

Thanks in advance for any answers. I did a search in the Discussions section for "dry hop," "dry hopping," "dry," and "hop," but came up with nothing.

**Comment by GnuUser** [comment]
Just FYI, I know that dry hopping produces no bitterness from AAs, but it apparently does from auxilliary bittering compounds (ABCs).

https://byo.com/article/ibu/

"Kettle vs. Dry Hops
Hops added to hot wort in the kettle undergo alpha acid isomerization, which produces the majority of bitterness in most beers. Dry hopping will produce no IAAs, but in large amounts it can produce significant bitterness from ABCs,3 especially from oxidized alpha acids created during hop storage. The IBUs from a dry hop addition are difficult to estimate, but the difference between adding hops to the boil kettle or to the conditioning vessel will have the largest impact on the IBU value."

**Comment by matty0ung** [comment]
See also https://github.com/Brewtarget/brewtarget/issues/774

**Reply by GnuUser** [reply]
Thank you!
